lib/migration.js

```diff
@@ -423,7 +432,7 @@ function mixinMigration(MySQL, mysql) {
         fkSQL = self.getForeignKeySQL(model, newFks);
     }
     if (fkSQL && fkSQL.length) {
-      self.applySqlChanges(model, fkSQL, function(err, result) {
+      self.applySqlChanges(model, [fkSQL.join(', ')], function(err, result) {
```

Comment:
@ssh24 please confirm replace `join` with `toString`.

btw, it should be more obvious to do
```javascript
self.execute(self.getAlterStatement(model, fkSQL), function(err, result) {
```
what do you think?